public/index.html

```diff
@@ -1,11 +1,15 @@
 <!doctype html>
 <html>
-  <head>
-    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
-<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-  </head>
-  <body>
-    <div id="photo"></div>
-    <script src="bundle.js"></script>
-  </body>
+
+<head>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
I don't think you should need React cdns if you're compiling them with Babel

Reply:
@IsaacLeeWebDev you are right, i took those cdns out